src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -178,25 +179,25 @@ class DepositServlet(app: EasyDepositApiApp)
   }
 
   private def respond(t: Throwable): ActionResult = (t match {
-    case e: IllegalStateTransitionException => Forbidden(e.getMessage)
+    case e: IllegalStateTransitionException => Forbidden(e.getMessage, Map(contentTypePlainText))
     case e: NoSuchDepositException => noSuchDepositResponse(e)
-    case e: NoSuchFileException => NotFound(body = s"${ e.getMessage } not found")
+    case e: NoSuchFileException => NotFound(body = s"${ e.getMessage } not found", Map(contentTypePlainText))
     case e: InvalidResourceException => invalidResourceResponse(e)
     case e: InvalidDocumentException => badDocResponse(e)
-    case e: ConcurrentUploadException => Conflict(e.getMessage)
-    case e: FileAlreadyExistsException => Conflict("Conflict. The following file(s) already exist on the server: "+e.getMessage)
-    case e: InvalidDoiException => BadRequest(e.getMessage)
-    case e: BadRequestException => BadRequest(e.getMessage)
-    case e: ZipMustBeOnlyFileException => BadRequest(e.getMessage)
-    case e: NotImplementedException => NotImplemented(e.getMessage)
-    case e: SchemaNotAvailableException => InternalServerError(e.getMessage)
+    case e: ConcurrentUploadException => Conflict(e.getMessage, Map(contentTypePlainText))
+    case e: FileAlreadyExistsException => Conflict("Conflict. The following file(s) already exist on the server: " + e.getMessage, Map(contentTypePlainText))
```

Comment:
Not sure if this is useful, but you can use interpolation here.

```suggestion
    case e: FileAlreadyExistsException => Conflict(s"Conflict. The following file(s) already exist on the server: ${ e.getMessage }", Map(contentTypePlainText))
```

Reply:
matter of taste